firefox-ios/firefox-ios-tests/Tests/XCUITests/DragAndDropTests.swift

```diff
@@ -21,10 +21,6 @@ let websiteWithSearchField = "https://developer.mozilla.org/en-US/"
 
 class DragAndDropTests: BaseTestCase {
 //  Disable test suite since in theory it does not make sense with Chron tabs implementation
-    override func tearDown() {
-        XCUIDevice.shared.orientation = UIDeviceOrientation.portrait
```

Comment:
We need to set the device to portrait again or the next tests would be run in landscape.

Reply:
@cavaevinci wanted to follow up on this PR, seems like there's one outstanding comment from @clarmso

Reply:
@cyndichin ,missed this one,sorry.I will resolve this in next 24 hours.

Reply:
no worries! thank you!